### DISCUSSION TITLE
Type error: Type '[IconPrefix, IconName]' is not assignable to type 'IconProp'.

### DISCUSSION THREAD

**Discussion by tigefa4u** [spencerwooo/onedrive-vercel-index#386] [Q&A] [answered]
![Screenshot 2022-02-09 at 14-19-35 Deployment Overview – Dashboard – Vercel](https://user-images.githubusercontent.com/1493571/153141011-bc4ea6d5-fe20-46cb-ad5f-89b48204de04.png)

**Comment by myl7** [comment, marked as answer]
See discussion #374 , you may need to change the build and install command in Vercel console.
